app/features/api-public/schema.ts

```diff
@@ -82,6 +82,8 @@ type SendouqMatchTeam = {
 
 type SendouqMatchPlayer = {
```

Comment:
Instead of adding fields here you could just query the /user/{identifier} endpoint